ui/admin/app/components/form/policy/policy-selection/index.js

```diff
@@ -25,7 +25,7 @@ export default class FormPolicySelectionComponent extends Component {
    * @type {boolean}
    */
   get isCustomRetentionSelected() {
-    return this.args.model.retain_for?.days > 0;
+    return this.args.model.retain_for?.days === 1;
```

Comment:
oops this is not right, pushing up a change, it should also exclude the  hippa and soc options